deployment-config.json

```diff
@@ -0,0 +1,56 @@
+{
+    "biomage-org": {
+        "domain_name": "Biomage",
+        "environment_names": [
+            "alabs",
+            "Biomage",
+            "trv"
+        ],
+        "docker_image_account": {
+            "id": 242905224710,
+            "region": "eu-west-1"
+        },
+        "support_email": "engineering@biomage.net",
+        "reply_email": "hello@biomage.net",
+        "deployment_environments": {
+            "trv": [
+                "production"
+            ],
+            "alabs": [
+                "production"
+            ],
+            "Biomage": [
+                "production",
+                "staging"
+            ]
+        },
+        "excluded_services": [
+            "cf/rds-agent-security-group.yaml",
+            "cf/rds-subnet-security-group.yaml"
+        ]
+    },
+    "hms-dbmi-cellenics": {
+        "domain_name": "HMS",
+        "environment_names": [
+            "HMS"
+        ],
+        "docker_image_account": {
+            "id": 160782110667,
+            "region": "us-east-1"
+        },
+        "support_email": "alex_pickering@hms.harvard.edu",
+        "reply_email": "alex_pickering@hms.harvard.edu",
```

Comment:
what is this used for?

Reply:
For Biomage we have two emails, one for support and one for reply so we had to make that abstraction even though for HMS its the same email.